src/components/CoreContents/CoreContents.js

```diff
@@ -1,14 +1,53 @@
-import React, { Component } from 'react'
+import React, { Component } from "react";
+import SubContents from "./SubContents/SubContents";
+import "./CoreContents.scss";
 
 class CoreContents extends Component {
-    render() {
-        return (
-            <div>
-                
-            </div>
-        )
-    }
-}
+  constructor() {
+    super();
+    this.state = {
+      mainItem: {},
+      subItems: [],
+    };
+  }
+
+  componentDidMount() {
+    fetch("http://localhost:3000/Data/CoreContentsdata.json")
+      .then((CoreContentsdata) => CoreContentsdata.json())
+      .then((contentsData) =>
+        this.setState({
+          mainItem: contentsData.mainItem,
+          subItems: contentsData.subItems,
+        })
+      );
```

Comment:
별건 아니지만 이렇게 줄일 수 있어요!
```suggestion
      .then((contentsData) =>{
       const {mainItem, subItems} = contentsData;
        this.setState({
          mainItem,
          subItems,
        })
        }
      );
```